site/gatsby-site/src/components/discover/Filter.js

```diff
@@ -7,10 +7,14 @@ import VirtualFilters from './VirtualFilters';
 import { Trans } from 'react-i18next';
 import { Accordion, Badge, Card } from 'flowbite-react';
 
-function ToggleContent({ label, touched, faIcon, toggled }) {
+function ToggleContent({ label, touched, faIcon, toggled, accordion = false }) {
   return (
-    <div className="flex flex-wrap items-center justify-between w-full">
-      <div className="flex text-xs items-center">
+    <div className="flex flex-nowrap items-center justify-between w-full">
+      <div
+        className={`flex items-center whitespace-nowrap ${
+          !accordion ? 'text-[0.8vw] md:text-[1vw] lg:text-xs' : 'text-xs'
```

Comment:
It still overflows sometimes:

![image](https://user-images.githubusercontent.com/25443411/234398942-7e292d01-b021-4a70-b2e8-1913a52b5ccc.png)

The text also look too small at the smallest size:

![image](https://user-images.githubusercontent.com/25443411/234399229-795f5828-916d-4c5c-acd2-07f7bb044856.png)

Reply:
Hey Luna, can you provide the screen size you're setting on the overlapping issue? I can't reproduce it. I fixed the font size. Thanks!

Reply:
It occurs for me at 1400px with the sidebar open, but investigating further it seems not to happen on a clean profile, so this is probably fine.